### ISSUE TITLE
Invalid syntax when assigning namespaced variables

### ISSUE THREAD

**Issue by rops** [sasstools/sass-lint#1335]
I'm trying to assign a namespaced variable to another var.

```
@use 'some/module` as m';

$foo: m.$bar;
```

This is building fine and behaving as expected, but sass-lint is triggering a parsing error.

```
error  Please check validity of the block starting from line #N  Fatal
```

Using `sass-lint@1.31.1`

**Comment by boubou158** [comment]
I am actually getting the same issue, any fix? thank you

**Comment by itsdevdom** [comment]
Duplicate of https://github.com/sasstools/sass-lint/issues/1305, check the last comment for a workaround
